app/views/users/service_providers/show.html.slim

```diff
@@ -8,29 +8,41 @@ h1 = t('headings.service_providers.show', name: service_provider.name)
 
 table.horizontal-headers
   tr
-    th Issuer
-    td.issuer = service_provider.issuer
-  tr
-    th Name
-    td.name = service_provider.name
+    th Friendly Name
+    td.friendly_name = service_provider.friendly_name
   tr
     th Description
     td.description = service_provider.description
   tr
-    th Metadata URL
-    td.metadata_url = service_provider.metadata_url
+    th Issuer
+    td.issuer = service_provider.issuer
+  tr
```

Comment:
I think we should merge this as-is and then create a new PR to refactor Agency into a model.